components/counter-bar/src/index.js

```diff
@@ -170,46 +208,60 @@ CounterBar.Total.displayName = 'Total';
 CounterBar.Total.propTypes = {
   active: PropTypes.bool,
   children: PropTypes.node,
+  component: PropTypes.node,
   countBackgroundColor: PropTypes.string,
   countDisabledBackgroundColor: PropTypes.string,
   score: PropTypes.number,
   scoreBackgroundColor: PropTypes.string,
   scoreDisabledBackgroundColor: PropTypes.string,
-  tag: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
 };
 
 CounterBar.Total.defaultProps = {
+  component: 'a',
   countBackgroundColor: WHITE,
   countDisabledBackgroundColor: BLACK,
   scoreBackgroundColor: RED,
   scoreDisabledBackgroundColor: GREY_3,
-  tag: 'a',
 };
 
-CounterBar.Item = ({active, children, score}) => {
- return (
-   <ItemWrapper
-     active={active}
-     disabled={!score}
-     empty={!children || children.length === 0}>
-     <ResultCountTitle
-       countColor={score > 0 ? WHITE : BLACK}
-       countBackgroundColor={score > 0 ? RED : GREY_3}
-       count={score || 0}
-          >
-       {children}
-     </ResultCountTitle>
-   </ItemWrapper>
- ); 
+CounterBar.Item = ({
+  active,
+  children,
+  component,
+  score,
+  ...props
+}) => {
+  const ItemWrapper = ItemWrapperInner.withComponent(component);
+  return (
+    <ItemWrapper
+      active={active}
+      disabled={!score}
+      empty={!children || children.length === 0}
+      {...props}
+    >
+      <ResultCountTitle
+        countColor={score > 0 ? WHITE : BLACK}
```

Comment:
these colors (including backgroundColor) need to be settable via props, much as they are for the total